sdcm/tester.py

```diff
@@ -1562,7 +1562,7 @@ def get_cluster_k8s_eks(self):  # pylint: disable=too-many-statements
         self.k8s_cluster.deploy_node_pool(
             eks.EksNodePool(
                 name=self.k8s_cluster.AUXILIARY_POOL_NAME,
-                num_nodes=2,
+                num_nodes=self.params.get('k8s_n_auxiliary_nodes'),
```

Comment:
Why only EKS? GKE also creates node pools of the same types

Reply:
added it also to GKE